elements/barcode_info.go

```diff
@@ -0,0 +1,6 @@
+package elements
+
+type BarcodeInfo struct {
+	DefaultModuleWidth int
+	DefaultHeight      int
```

Comment:
```suggestion
	ModuleWidth int
	Height      int
```
most likely we can just omit the word Default from all properties and instead add it to the variable that contains the struct

Reply:
ok